app/src/main/java/com/example/concordia_campus_guide/Activities/SearchActivity.java

```diff
@@ -66,6 +80,26 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
                 openRoutesPage(place);
             }
         });
+
+        final CalendarEvent calendarEvent = calendarViewModel.getEvent(getApplicationContext(), this);
+
+        nextClassText.setText(calendarViewModel.getNextClassString(calendarEvent));
+
+        if(nextClassText.getText() != "No classes today"){
+            nextClassArrow.setVisibility(View.VISIBLE);
+            nextClassRow.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    openRoutesPage(new Place() {
```

Comment:
This doesn't pass the actual place ie: with all attributes, maybe getting from the database, I think there should be a way to figure out how to pass the actual place as part of this story

Reply:
I will be passing a room (which is a Place), is that sufficient? Let me know!

Reply:
yeah thats good, but i think it would be better to pass a classroom with the full information, rather than just the display name because we will need that